packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/activeDirectories/delete/federation-delete.controller.js

```diff
@@ -0,0 +1,46 @@
+export default class ActiveDirectoriesDeleteController {
+  /* @ngInject */
+  constructor($translate, DedicatedCloud) {
+    this.$translate = $translate;
+    this.DedicatedCloud = DedicatedCloud;
```

Comment:
it's a good practice to name the class instance with minus first letter.
```suggestion
    this.dedicatedCloud = DedicatedCloud;
```